docs/hub/api.md

```diff
@@ -6,15 +6,15 @@ We have open endpoints that you can use to retrieve information from the Hub as
-| /api/metrics     GET                                                         	| Get information from all metrics in the Hub.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           	| `list_metrics()`                  	|                                                                                                                                                                       	|   	|
+| /api/metrics     GET                                                         	| Get information from all metrics in the Hub. The response is paginated, use the [`Link` header](https://docs.github.com/en/rest/guides/using-pagination-in-the-rest-api?apiVersion=2022-11-28#link-header) to get the following pages.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           	| `list_metrics()`                  	|                                                                                                                                                                       	|   	|
```

Comment:
I don't think this endpoint is paginated. It only returns 115 items :)

(suggestion is just to revert it back without changes)

```suggestion
| /api/metrics     GET                                                         	| Get information from all metrics in the Hub.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           	| `list_metrics()`                  	|                                                                                                                                                                       	|   	|
```

Reply:
You're right, I just checked in the source code, it's not paginated.